var/spack/repos/builtin/packages/arbor/package.py

```diff
@@ -6,68 +6,67 @@
 from spack import *
 
 
-class Arbor(CMakePackage):
+class Arbor(CMakePackage, CudaPackage):
     """Arbor is a high-performance library for computational neuroscience
     simulations."""
 
     homepage = "https://github.com/arbor-sim/arbor/"
-    url      = "https://github.com/arbor-sim/arbor/archive/v0.2.tar.gz"
+    git      = "https://github.com/arbor-sim/arbor.git"
+    maintainers = ['bcumming', 'halfflat']
 
-    version('0.4', sha256='7d9fc6b3262954cc5dc1751215fbb9f2cdb7010e829a2be43022f90da2d8d2e3')
-    version('0.2', sha256='43c9181c03be5f3c9820b2b50592d7b41344f37e1200980119ad347eb7bcf4eb')
+    version('master', branch='master')
+    version('0.5', commit="2158d2c61c0608363c66b9665ca4785e8c290b37", submodules=True)
```

Comment:
Do you support external dependencies for what you currently have in your submodules? I see `py-bind11` is already listed below and `benchmark` is wrapped already (but not listed here).

Reply:
We do. Se explicitly made it possible to use external dependencies instead of our bundled sub-modules by default, to make spack integration better.

The benchmark sub module is not required for the release build, python target or docs (the devs use it for micro benchmarks)

Reply:
In that case it might be better to use tarballs, cause it saves a git clone and it allows spack to verify the checksum

Reply:
done